terraform/catalogue_api/terraform.tf

```diff
@@ -0,0 +1,47 @@
+terraform {
+  backend "s3" {
+    role_arn = "arn:aws:iam::756629837203:role/catalogue-developer"
+
+    bucket         = "wellcomecollection-catalogue-infra-delta"
+    key            = "terraform/catalogue/api/catalogue_api.tfstate"
```

Comment:
Minor: the "catalogue" repo is no more, let's update this key to match:

```suggestion
    key            = "terraform/catalogue-api/catalogue_api.tfstate"
```

Reply:
Very neat that terraform can tell I've done this change and offers to copy across the state from the previous backend